doc/user/content/manage/user-management/rbac-newrole.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Create a new user and role"
+description: ""
+menu:
+  main:
+    parent: user-management
+    weight: 15
+---
+
+This guide walks you through creating a new user and new role in Materialize. By
+the end of this tutorial you will:
+
+* Invite a new user in the Materialize Console
+* Create a new role in your Materialize account using `psql`
+* Apply privileges to the new role
+* Assign the new role to the new user
+
+## Before you begin
+
+* Make sure you have `psql` installed locally.
+
+* Make sure you have a [Materialize account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
+
+## Step 1. Invite a new user
+
+First, [login to the Materialize console](https://console.materialize.com/) and navigate to Account > Account
+Settings > Users.
+
+Click **Invite User** and fill in the user information.
+
+The **Organization Admin** and **Organization Member** roles refer to `SUPERUSER`
+privileges
+
+## Step 2. Connect to Materialize
+
+Materialize stores role names, IDs, and object IDs in the `mz_roles` catalog
+table. You need to connect to your Materialize instance with `psql` to add roles
+to this table.
+
+1. In the [Materialize UI](https://console.materialize.com/), go to the **Connect** screen, create a new app password and then copy the `psql` command.
+
+    The app password will be displayed only once, so be sure to copy the password somewhere safe. If you forget your password, you can create a new one.
+
+## Step 3. Create a new role
+
+1. Open a new terminal window, run the `psql` command, and enter your app password.
+
+    Now that you're logged in, you can create a new role in the `mz_roles`
+    table.
+
+    ```sql
+    CREATE ROLE dev_role;
+    ```
+
+1. Each role you create has default **role attributes** that determine how they
+    can interact with Materialize objects. Let's look at the role attributes of
+    the role you created:
+
+    ```sql
+    SELECT * FROM mz_roles WHERE 'name' = dev_role;
+    ```
+
+    Your `dev_role` returns attributes similar to the following:
+
+    ```nofmt
+    -[ RECORD 1 ]--+------
+    id             | u8
+    oid            | 50991
+    name           | dev_role
+    inherit        | t
+    create_role    | f
+    create_db      | f
+    create_cluster | f
+    ```
+    Your `id` and `oid` values will look different.
+
+    The `inherit`, `create_role`, `create_db`, and `create_cluster` are the
+    role attributes assigned to a role when it is created. These attributes
+    determine the system-level permissions of a role and do not impact object-level privileges.
+
+   * `INHERIT` is set to true by default and allows roles to inherit the
```

Comment:
This isn't rendering right:
<img width="764" alt="Screenshot 2023-05-18 at 10 16 26 AM" src="https://github.com/MaterializeInc/materialize/assets/15893490/d5b25168-21bf-4e77-8baa-1a25b1d4c268">